docs/reference/experiment-config.txt

```diff
@@ -472,21 +467,21 @@ created with the Determined CLI. It may contain the following fields:
 .. _batches-per-step:
 
 - ``batches_per_step``: Specifies the number of batches in a single
```

Comment:
Are we not renaming `batches_per_step`?

Reply:
This is coming in a follow up change right after this, mainly to simplify #937 just a little bit since it's already a large change.